Calculator/app/.gitignore

```
@@ -0,0 +1 @@
+/build
```

Add this to your `.gitingnore`
```suggestion
/build
/.idea/*
```